c_binding/include/libstoragemgmt/libstoragemgmt_nfsexport.h

```diff
@@ -128,19 +205,47 @@ int LSM_DLL_EXPORT lsm_nfs_export_export_path_set(lsm_nfs_export *exp,
                                                   const char *export_path);
 
 /**
- * Returns the client authentication type
- * @param exp       Valid nfs export record
- * @return Pointer to authentication type
+ * lsm_nfs_export_auth_type_get - Retrieves the authentication type of the
+ * nfs_export
+ *
+ * Version:
+ *      1.0
+ *
+ * Description:
+ *      Retrieves the authentication type for the lsm_nfs_export.
+ *      Note: Address returned is valid until lsm_nfs_export gets freed, copy
+ *      return value if you need longer scope. Do not free returned string.
+ *
+ * @exp:
+ *      NFS export to retrieve authentication type for.
+ *
+ * Return:
+ *      string. NULL if argument 'exp' is NULL or not a valid lsm_nfs_export
+ *      pointer.
  */
 const char LSM_DLL_EXPORT *lsm_nfs_export_auth_type_get(lsm_nfs_export
                                                         * exp);
 int LSM_DLL_EXPORT lsm_nfs_export_auth_type_set(lsm_nfs_export *exp,
```

Comment:
Again missing doc for `lsm_nfs_export_auth_type_set` but wondering if it should even be in this file.